flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/databases/oracle/dialect/OraclePreparedStatementTest.java

```diff
@@ -48,10 +48,10 @@ void testInsertStatement() {
         String insertStmt = dialect.getInsertIntoStatement(tableName, fieldNames);
         assertThat(insertStmt)
                 .isEqualTo(
-                        "INSERT INTO tbl(id, name, email, ts, field1, field_2, __field_3__) "
+                        "INSERT INTO \"tbl\"(\"id\", \"name\", \"email\", \"ts\", \"field1\", \"field_2\", \"__field_3__\") "
                                 + "VALUES (:id, :name, :email, :ts, :field1, :field_2, :__field_3__)");
         NamedStatementMatcher.parsedSql(
-                        "INSERT INTO tbl(id, name, email, ts, field1, field_2, __field_3__) "
+                        "INSERT INTO \"tbl\"(\"id\", \"name\", \"email\", \"ts\", \"field1\", \"field_2\", \"__field_3__\") "
```

Comment:
In issue description it was only something about table name
why does it impact columns?

Reply:
the method ‘quoteIdentifier’ is common，column metadata is also by call the method.Columns should have this problem as well

Reply:
can we have a a filaing test reproducing the problem?